packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -104,7 +104,7 @@ export class CdkToolkit {
       // Compare N stacks against deployed templates
       for (const stack of stacks.stackArtifacts) {
         stream.write(format('Stack %s\n', chalk.bold(stack.displayName)));
-        const currentTemplate = await this.props.cloudFormation.readCurrentTemplate(stack);
+        const currentTemplate = await this.props.cloudFormation.readCurrentTemplateWithNestedStacks(stack); // TODO; renmae
```

Comment:
Why the ToDo here? The name looks good to me...?

Reply:
I mistakenly thought you were referring to this function as well in an earlier comment, nvm.

Reply:
Can we remove the ToDo then...? 🙂